package.json

```diff
@@ -21,7 +21,7 @@
   "author": "Braintree <code@getbraintree.com> (https://www.braintreepayments.com/)",
   "license": "MIT",
   "devDependencies": {
-    "@types/jest": "^29.5.2",
+    "@types/jest": "^29.4.0",
```

Comment:
curious why we had to downgrade the version of jest, here, but I also know this is still a work in progress

Reply:
It should still be the original version we already used. I uninstalled it when removing the rest of Jest dependencies, but then found it was still necessary to include for type definitions so I re-installed it.